src/ontology/cl-edit.owl

```diff
@@ -24973,8 +24973,13 @@ EquivalentClasses(obo:CL_1000222 ObjectIntersectionOf(obo:CL_0000165 ObjectSomeV
 
 # Class: obo:CL_1000223 (lung neuroendocrine cell)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33355253") Annotation(oboInOwl:hasDbXref "doi:10.3109/01902148209069653") obo:IAO_0000115 obo:CL_1000223 "A neuroendocrine cell that is part of respiratory epithelium of the lung and can detect environmental stimuli, including hypoxic conditions through chemoreception. Ultrastructurally, this cell type is characterized by the presence of cytoplasmic dense core granules, which are considered the storage site of amine and peptide hormones. Lung neuroendocrine cells appear as solitary cells or as clustered masses, localized at airway bifurcation sites, called neuroepithelial bodies that can release serotonin in response to hypoxia. The lung neuroendocrine cell type is also reported to have stem cell properties.")
```

Comment:
Consider changing:
... considered the storage site of amine... --> ... considered the storage sites of amine..